script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -607,157 +625,163 @@ Arcueid?!"
 [sha:0705c22cb4991b932b6affde19ea44889321fa24]{
 -- Page 542, Offset 16103.
 -- 振り返った先に白い姿はなかった。
-When I turn around, her white figure is gone.
+When I turn around, the white figure that had been there a moment ago is nowhere to be seen.
 }
 [sha:232f07d0e64d1e3465301fcaeffdcc02891b97ed]{
 -- Page 542, Offset 16104.
 -- 俺がシャッターを切っている間に、アルクェイドはこの場から消えていた。
-As I was cutting the shutter, Arcueid disappeared from the scene.
+Arcueid vanished while I was busy cutting the shutter.
 }
 [sha:27937e420c451848df02430ca3a97af40a2dd488]{
 -- Page 543, Offset 16105.
 -- 「ま―――」
-"Cr————"
+"Cr———"
 }
 [sha:fa5a1a44ff39fa362803d4dbd28d3ebb98d37db7]{
 -- Page 543, Offset 16106.
 -- 気がついた時には、もう取り返しの付かない死地にいた。
-And before I know it, I stand irrevocably before the jaws of death.
+Without realizing it, I've wandered straight into the inescapable jaws of death.
 }
 [sha:33729d9fd9f470579e96f5db0bc477af35750688]{
 -- Page 543, Offset 16107.
 -- 一歩も下がれない。
-I cannot retreat even a single step.
+I can't retreat at all.
 }
 [sha:c5212cbe93b4073d7073d041fd4b77040f420216]{
 -- Page 543, Offset 16108.
 -- 振り返る事もできない。
-I also cannot look back.
+I can't even turn to look back.
 }
 [sha:2b64271c4313823551ff80a6b2112514a1330469]{
 -- Page 543, Offset 16109.
 -- <ヤ|・><ツ|・><ラ|・>から視線を切った瞬間、俺は無様な<生贄|くいもの>になる。
 // ruby cut.
-As soon as I take my gaze away from %{i}them%{/i}, I will be reduced to an unsightly sacrifice.
+The instant I take my gaze off %{i}them%{/i}, I'll be reduced to an unsightly sacrifice.
 }
 [sha:3144ee796ea94bec2f98419c586eba265cedec87]{
 -- Page 544, Offset 16110.
 -- 「は―――、ぁ」
-"Hahhh..."
+"%{i}Ugh%{/i}... %{i}ah%{/i}―――"
 }
 [sha:2fc6212fa6ad6da7c1d611633dc14331c39c9b25]{
 -- Page 544, Offset 16111.
 -- 視界が朱く断裂する。
-My vision ruptures into crimson.
+Something snaps, and my vision flashes crimson.
```

Comment:
Yeah this is a shitty sentence to translate. "My field of vision snaps _redly_"? That's not an adverb buddy. Looking at this again I'd probably ascribe the action _to_ the crimson, i.e.
> Crimson ruptures my vision.
> Crimson slashes across my vision.

instead of just massaging it to fit with the in-game VFX. The old TL is definitely _interesting_, but such translations are a gamble. Interesting can be as much an insult as a compliment. I think that here it might be OK though, since the line after is also some wacky abstract stuff. If the old TL is the preferred of the 3 I'll revert.

Reply:
The line is strange in Japanese too, in every sense of the word. 視界s don't 断裂, and they sure as well don't do so 朱く.
I feel like that character of it should very much be preserved. Making it boring to be safe just feels like the completely wrong attitude.